### PR TITLE
Add changelog entry for v2 alpha release

### PR DESCRIPTION
**Pull request by samuelhwilliams**

### CODE REVIEW

File: doc/changelog.rst [hunk 1]
```diff
@@ -1,6 +1,27 @@
 Changelog
 =========
 
+2.0.0a0
+-------
+
+Breaking changes:
+* Added support for Python 3.12
```
Comment (princerb) [inline doc/changelog.rst:8]: * Update for compatibility with Flask 3.
* Update for compatibility with WTForms 3.

File: doc/changelog.rst [hunk 2]
```diff
@@ -1,6 +1,27 @@
...
+* Dropped support for Python 3.7
+* Flask-BabelEx is no longer supported; the package is no longer maintained and Flask-Babel is recommended/active instead.
+* Flask-Mongoengine is no longer supported due to that package being unmaintained.
+* Bootstrap2 and Bootstrap3 themes are no longer available.
+* All Flask-Admin config has been namespaced under `FLASK_ADMIN_`.
```
Comment (princerb) [inline doc/changelog.rst:13]: * Remove the usage of the config `FLASK_ADMIN_SWATCH` (caused by `Admin()` instance's new parameter - `theme`)

**Review by princerb — COMMENTED** [review]
What about adding these too?

File: doc/changelog.rst (hunk 1 above)
Comment (samuelhwilliams) [reply doc/changelog.rst:8]: These aren't breaking changes, and are already listed further down 👍

File: doc/changelog.rst (hunk 2 above)
Comment (samuelhwilliams) [reply doc/changelog.rst:13]: Good call, will tweak wording.

File: doc/changelog.rst [hunk 3]
```diff
@@ -11,6 +11,7 @@ Breaking changes:
 * Flask-Mongoengine is no longer supported due to that package being unmaintained.
 * Bootstrap2 and Bootstrap3 themes are no longer available.
 * All Flask-Admin config has been namespaced under `FLASK_ADMIN_`.
+  * `FLASK_ADMIN_SWATCH` has been removed; see `theme` parameter below.
 * `Admin()` now takes a `theme` parameter that encapsulates all of the configuration options for theming the admin instance. This replaces the `template_mode` parameter.
 
 New features:
```
Comment (princerb) [inline doc/changelog.rst:17]: I guess it needs a new empty line to fix the preview correctly?

**Pull request merged**